src/sentry/static/sentry/app/views/issueList/sidebar.tsx

```diff
@@ -22,7 +22,6 @@ type DefaultProps = {
 };
 
 type Props = DefaultProps & {
-  orgId: string;
```

Comment:
orgId is passed from overview -> sidebar -> tagfilter

but it seems like tagFilter doesn't even use orgId so it is now removed